jobbergate-api/jobbergate_api/tests/apps/conftest.py

```diff
@@ -92,6 +92,7 @@ def job_submission_data():
         "job_submission_owner_email": "owner1@org.com",
         "client_id": "dummy-client-id",
         "status": JobSubmissionStatus.CREATED,
+        "execution_parameters": {"name": "job-submission-name", "comment": "I am a comment"},
```

Comment:
Including this column on the `job_submission_data` fixture, so it propagates to all the tests.